### PR TITLE
Enhanced ActsAsMappable#in_bounds to allow the query to include thing…

### PR DESCRIPTION
**Pull request by denisahearn**
…s that are located on the rectangular bounds, via an "inclusive" option (defaults to false).

### CODE REVIEW

File: test/acts_as_mappable_test.rb [hunk 1]
```diff
@@ -389,6 +389,16 @@ def test_find_within_bounds
     assert_equal 2, locations.count
   end
 
+  def test_find_within_bounds_with_inclusive
+    sw = Geokit::LatLng.new(@loc_a.lat,@loc_a.lng)
+    ne = sw
+    bounds = [sw,ne]
+    locations = Location.in_bounds(bounds, :inclusive => false)
+    assert_equal 0, locations.count
+    locations = Location.in_bounds(bounds, :inclusive => true)
```
[the remark this answers is not in the document]
Comment (denisahearn) [reply test/acts_as_mappable_test.rb]: Normally I would, but I followed the style in the existing code.

File: test/acts_as_mappable_test.rb [hunk 2]
```diff
@@ -389,6 +389,16 @@ def test_find_within_bounds
     assert_equal 2, locations.count
   end
 
+  def test_find_within_bounds_with_inclusive
+    sw = Geokit::LatLng.new(@loc_a.lat,@loc_a.lng)
+    ne = sw
+    bounds = [sw,ne]
```
[the remark this answers is not in the document]
Comment (denisahearn) [reply test/acts_as_mappable_test.rb]: Normally I would, but I followed the style in the existing code.

File: test/acts_as_mappable_test.rb [hunk 3]
```diff
@@ -389,6 +389,16 @@ def test_find_within_bounds
...
+    locations = Location.in_bounds(bounds, :inclusive => false)
```
[the remark this answers is not in the document]
Comment (denisahearn) [reply test/acts_as_mappable_test.rb]: Normally I would, but I followed the style in the existing code.

File: test/acts_as_mappable_test.rb [hunk 4]
```diff
@@ -389,6 +389,16 @@ def test_find_within_bounds
     assert_equal 2, locations.count
   end
 
+  def test_find_within_bounds_with_inclusive
+    sw = Geokit::LatLng.new(@loc_a.lat,@loc_a.lng)
```
[the remark this answers is not in the document]
Comment (denisahearn) [reply test/acts_as_mappable_test.rb]: Normally I would, but I followed the style in the existing code.

**Comment by mnoack** [comment]
@denisahearn - Sorry I haven't responded timely. I think I'll need to find help amongst people contributing to review PRs going forward and I no longer use geokit so it's best to be done by those using geokit.

So if you were available to occasionally review geokit PRs (given they'll affect you, generally positively as the tests are fairly comprehensive) then please accept the invite to become a committer and feel free to merge this PR.

**Comment by denisahearn** [comment]
@mnoack Thanks for responding, I've accepted your invitation. Yes, I would be willing to help review PRs for `geokit-rails`. I will merge my PR. Do you plan to still be the person to release new versions of gem? It looks like there are a few commits to master since version `2.3.1` was released last May, and I'm curious to know when a new version with my commit will be released to rubygems.org.

**Comment by mnoack** [comment]
You're welcome to release gems, I think the process was a standard process like updating the version.rb file and then "bundle exec rake release" or something, feel free to "rake -T" and look for the obvious one.

My general approach was not to release after every PR, but generally as people requested it, as after-all people can point their gemfiles to the github master, this way I felt it got a bit of "testing" that way before publishing, but it really is your call now.

**Pull request merged**